code/game/objects/effects/effect_system/smoke.dm

```diff
@@ -257,6 +257,12 @@
 	color = "#DBCBB9"
 	smoke_traits = SMOKE_GASP|SMOKE_BLISTERING|SMOKE_OXYLOSS|SMOKE_PLASMALOSS|SMOKE_FOUL
 
+/obj/effect/particle_effect/smoke/phosphorus/mustard
+	opacity = TRUE
+	color = "#FFDB58"
+	lifetime = 40
```

Comment:
A rather indecise lifetime, not a good fit for a shock role or an area suppresion role, 20 or 60 would be better picks depending on what you wan to do with it